Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -1066,7 +1066,7 @@ class Gx2Fitter {
 
     // This will be filled during the updates with the final covariance of the
     // track parameters.
-    BoundMatrix fullCovariancePredicted = BoundMatrix::Identity();
+    BoundMatrix fullCovariance = BoundMatrix::Identity();
```

Comment:
Same as above, we need to specify, that it is the Covariance of the (fitted) parameters.